mongodbatlas/alert_configurations.go

```diff
@@ -0,0 +1,309 @@
+package mongodbatlas
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+const alertConfigurationPath = "groups/%s/alertConfigs"
+
+// AlertConfigurationsService is an interface for interfacing with the Project IP Whitelist
+// endpoints of the MongoDB Atlas API.
+// See more: hhttps://docs.atlas.mongodb.com/reference/api/alert-configurations
+type AlertConfigurationsService interface {
+	Create(context.Context, string, *AlertConfiguration) (*AlertConfiguration, *Response, error)
+	EnableAnAlert(context.Context, string, string, *bool) (*AlertConfiguration, *Response, error)
+	GetAnAlert(context.Context, string, string) (*AlertConfiguration, *Response, error)
+	GetOpenAlerts(context.Context, string, string) ([]AlertConfiguration, *Response, error)
```

Comment:
Since there are a little different let's ensure the name is clear.  For example, GetAnAlert doesn't get an actual Alert but the config so naming it GetAnAlertConfig would be clearer imo.   Same with EnableAnAlertConfig and Get OpenAlertsConfigs.  This is just to ensure this isn't confused with the Alerts Resource (https://docs.atlas.mongodb.com/reference/api/alerts/)

Reply:
okay make sense, thanks